.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+    "env": {
+        "es6": true,
+        "node": true
```

Comment:
You might need to add https://github.com/fabric8-hdd/manager-api/pull/2/files#diff-df39304d828831c44a2b9f38cd45289cR5
to suppress undesirable mocha related linter complaints